### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by renancaraujo**
Add github action workflow running both format and test.

### CODE REVIEW

**Comment by erickzanardo** [comment]
Build has failed @renancaraujo looks like the ./format.sh file can't be found.

**Comment by renancaraujo** [comment]
Forgot to update the path

**Comment by renancaraujo** [comment]
Now it goes.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
```
Comment (luanpotter) [inline .github/workflows/test.yml]: do we need java?

**Comment by erickzanardo** [comment]
anything preventing this from been merged @renancaraujo ?

**Pull request merged**